src/app_engine/probers.py

```diff
@@ -12,13 +12,18 @@
 import json
 import numbers
 import webapp2
+import sys
 from google.appengine.api import mail
+from google.appengine.api import memcache
 from google.appengine.api import urlfetch
 
-def send_alert_email(tag, message):
+def send_alert_email(tag, message, instance=None):
   receiver = 'tachyon-eng@google.com'
   sender_address = 'AppRTC Notification <jiayl@google.com>'
-  subject = 'AppRTC Prober Alert: ' + tag
+  subject = 'AppRTC Prober Alert'
+  if (instance != None):
+    subject += ' (' + instance + ') '
```

Comment:
Or in the 'message'. I'd prefer to avoid extra argument that is not common to CEOD and collider.